test/basic.html

```diff
@@ -40,8 +40,12 @@
         var pages;
 
         suite('defaults', function() {
-          setup(function () {
+          setup(function(done) {
             pages = fixture('basic');
+            pages.addEventListener('iron-items-changed', function onIronItemsChanged() {
```

Comment:
Same problem as mentioned in [this thread](https://github.com/PolymerElements/iron-selector/pull/139#discussion_r109078246) of iron-selector.